doc/api/util.md

```diff
@@ -269,7 +269,7 @@ function MyStream() {
 
 util.inherits(MyStream, EventEmitter);
 
-MyStream.prototype.write = function(data) {
+MyStream.prototype.write = (data) => {
   this.emit('data', data);
```

Comment:
This is incorrect. You **never** want to use arrow functions on the prototype.

`this.emit()` now uses the wrong `this`.